k8s/overlays/azure/test/kustomization.yaml

```diff
@@ -77,6 +79,14 @@ patches:
               - name: cse-import-d2cc-runner-itools-configmap-volume
                 configMap:
                   name: cse-runner-itools-azure-test-configmap
+  # Scale-up to 4 instances and adapt CSE runner itools config
```

Comment:
Please check that the computation for one task will not be runned 4 times

Reply:
done